src/platform/user/profile/vap-svc/components/base/VAPServiceEditModalErrorMessage.jsx

```diff
@@ -68,21 +63,20 @@ export default function VAPServiceEditModalErrorMessage({
     default:
       content = (
         <p>
-          We’re sorry. We can’t save your {title.toLowerCase()} at this time.
-          We’re working to fix this problem. Please try again or check back
-          soon.
+          We're sorry. We can't update your information right now. We're working
```

Comment:
All these straight single quotes should be curly quotes, which you can get by typing `⌥ ]` if you are on a Mac. We have an ESLint rule that should be flagging these for you, but I confirmed that it doesn't seem to be working 🤔  😞